examples/apply-setters-simple/README.md

```diff
@@ -2,8 +2,8 @@
 
 ### Overview
 
-The `apply-setters` KRM config function applies setter values to resource fields
-with setter references.
+The `apply-setters` function applies setter values to resource fields
```

Comment:
Examples shouldn't repeat definition of setters. That's already covered in usage doc.

Reply:
Updated.